protocol.html

```diff
@@ -688,6 +688,29 @@ <h2 property="schema:name">Notifications</h2>
               <p>A Solid server MUST conform to the LDN specification by implementing the Receiver parts to receive notifications and make Inbox contents available [<cite><a class="bibref" href="#bib-ldn">LDN</a></cite>].</p>
```

Comment:
"by implementing the Receiver part" - the receiver part of the spec I guess. It's a bit oddly phrased. 
Can on not just say: A Solid server must have an ldn:inbox to which notifications can be sent? 

"And make inbox content available" -  To whom?

Reply:
Happy to revise outside of this PR, but just to respond:

It is not that "a Solid server must have an ldn:inbox.." but that server needs to conform to LDN's requirements (receiver) for inbox resources.

Reply:
I am happy with this introspection pattern for enabling User Agents deduce the locations of special resources e.g., inbox folders.